File: CS/MyCardView/MyCardInfoRegistrator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Registrator;
using DevExpress.XtraGrid.Views.Base;
using DevExpress.XtraGrid.Views.Base.Handler;
using DevExpress.XtraGrid.Views.Base.ViewInfo;

namespace MyCardView
{
    public class MyCardInfoRegistrator : CardInfoRegistrator
    {
        public override string ViewName { get { return "MyCardView"; } }
        public override BaseView CreateView(GridControl grid) { return new MyCardView(grid as GridControl); }
        public override BaseViewInfo CreateViewInfo(BaseView view){ return new MyCardViewInfo(view as MyCardView); }
        public override BaseViewHandler CreateHandler(BaseView view) { return new MyCardHandler(view as MyCardView); }
    }
}

File: VB/MyCardView/MyCardInfoRegistrator.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Registrator
Imports DevExpress.XtraGrid.Views.Base
Imports DevExpress.XtraGrid.Views.Base.Handler
Imports DevExpress.XtraGrid.Views.Base.ViewInfo

Namespace MyCardView
	Public Class MyCardInfoRegistrator
		Inherits CardInfoRegistrator

		Public Overrides ReadOnly Property ViewName() As String
			Get
				Return "MyCardView"
			End Get
		End Property
		Public Overrides Function CreateView(ByVal grid As GridControl) As BaseView
			Return New MyCardView(TryCast(grid, GridControl))
		End Function
		Public Overrides Function CreateViewInfo(ByVal view As BaseView) As BaseViewInfo
			Return New MyCardViewInfo(TryCast(view, MyCardView))
		End Function
		Public Overrides Function CreateHandler(ByVal view As BaseView) As BaseViewHandler
			Return New MyCardHandler(TryCast(view, MyCardView))
		End Function
	End Class
End Namespace
